sci-biology/open-ephys-gui/open-ephys-gui-99999999.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake
+
+DESCRIPTION="Processing, recording, and visualizing multichannel ephys data"
+HOMEPAGE="http://www.open-ephys.org/gui/"
+LICENSE="GPL-3"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/open-ephys/plugin-GUI"
+	EGIT_BRANCH="master"
+    inherit git-r3
+	Suffix=$EGIT_BRANCH
+	SubDir=${P}
+elif [[ ${PV} == "99999999" ]] ; then
+	EGIT_REPO_URI="https://github.com/open-ephys/plugin-GUI"
+	EGIT_BRANCH="development"
+    inherit git-r3
+	Suffix=$EGIT_BRANCH
+	SubDir=${P}
+else
+    SRC_URI="https://github.com/open-ephys/plugin-GUI/archive/v${PV}.tar.gz"
+	Suffix=${PV}
+	SubDir="plugin-GUI-${PV}"
+	S="${WORKDIR}/$SubDir"
+fi
+
+SLOT="${PV}"
+KEYWORDS="~amd64 ~x86"
```

Comment:
please remove keywords from 9999 and above

Reply:
Done